src/main/scala/singleton/ops/impl/GeneralMacros.scala

```diff
@@ -1381,6 +1437,7 @@ trait GeneralMacros {
           else genOpTreeNat(opTpe, t)
         case (_, CalcLit(_, t)) => genOpTreeLit(opTpe, t)
         case (funcTypes.AcceptNonLiteral | funcTypes.GetArg, t : CalcNLit) => genOpTreeNLit(opTpe, t)
+        case (_, t @ CalcUnknown(_,_,true)) => genOpTreeUnknown(opTpe, t)
```

Comment:
this new case looks like it executes exactly the same call to genOpTreeUnknown as the following one

Reply:
No, the following one is limited to the `GetArg` functionality